lib/Models/WebProcessingServiceCatalogFunction.js

```diff
@@ -53,6 +56,13 @@ function WebProcessingServiceCatalogFunction(terria) {
      */
     this.identifier = undefined;
 
+    /**
+     * Gets or sets whether to use key value pairs (KVP) embedded in Execute URL, or whether to make a POST request
+     * with XML data.
+     * @type {Boolean}
+     */
+    this.useKvp = undefined;
```

Comment:
We should document the default.  Also, is there a better name for this?  The spec describes it as "Execute HTTP GET request KVP encoding" (versus "Execute HTTP POST request XML encoding").  How about `executeWithHttpGet`?

You probably already know this, but I was interested to learn that WPS servers are required to support POST for Execute, but GET is optional.  So I think changing the default to POST makes a lot of sense.